pkg/api/types.go

```diff
@@ -269,3 +269,22 @@ type GeneralStageStatus struct {
 	StartTime time.Time `bson:"startTime,omitempty" json:"startTime,omitempty" description:"start time of the stage"`
 	EndTime   time.Time `bson:"endTime,omitempty" json:"endTime,omitempty" description:"end time of the stage"`
 }
+
+// ListMeta represents metadata that list resources must have.
+type ListMeta struct {
+	Total       int `json:"total,omitempty" description:"total items count"`
+	ItemsLength int `json:"itemsLength,omitempty" description:"returned items count"`
```

Comment:
Both `Total` and `ItemsLength` is required, can not be `omitemtpy`. `Items` can be `omitemtpy`.